src/IRac.cpp

```diff
@@ -3304,12 +3307,12 @@ int16_t IRac::strToModel(const char *str, const int16_t def) {
 /// @return The equivalent boolean value.
 bool IRac::strToBool(const char *str, const bool def) {
   if (!STRCASECMP(str, kOnStr) ||
-      !STRCASECMP(str, F("1")) ||
+      !STRCASECMP(str, PSTR("1")) ||
       !STRCASECMP(str, kYesStr) ||
       !STRCASECMP(str, kTrueStr))
     return true;
   else if (!STRCASECMP(str, kOffStr) ||
-           !STRCASECMP(str, F("0")) ||
+           !STRCASECMP(str, PSTR("0")) ||
```

Comment:
How large is this address? Is it maybe cheaper to store this one byte as is? (Not saying we should, just trying to see corner cases)

Reply:
Good point. I'll look into it
Though, technically it is two bytes. `"0"` is `{0x30, 0x00}`

So, if the alignment is a 16bit DWORD, we should be fine, if 32bit, then yeah, it is wasted space.

Reply:
On 0 and 1 there is no need for casecmp, if that helps.

Reply:
Looks like DWORDs are 32bit. Thus we are wasting 2 bytes of flash.
The other option is it is stored in ram (not 100% sure on this), thus we lose 2 bytes of a much more finite resource.

Reply:
> On 0 and 1 there is no need for casecmp, if that helps.

Yeah, I figured that also. I'm not sure it helps.

Reply:
FWIW, rough calc of wasted space:
_There are approx 170+ flash-stored strings. Rounding up to 200 for simplicity/safety.
 Worst case alignment scenario is 3 bytes wasted per string. So 600 bytes wasted.
 Average case (most likely) is 2 bytes wasted, so 400 bytes._

If we packed them like the protocol names and traded off cpu & memory cycles, plus loss of code niceness/flexibility, we could lose only 4 bytes total (worst case) plus additional code overhead.

If we were saving IRAM, then yes it'd be worth it; Ram, probably; Flash, probably not.

We eat about 1-2k of flash per additional protocol, so not that worried for 400-ish bytes of flash. Yet.